images/win/scripts/ImageHelpers/VisualStudioHelpers.ps1

```diff
@@ -61,20 +61,57 @@ Function Install-VisualStudio
 }
 
 function Get-VsCatalogJsonPath {
-    $instanceFolder = Get-Item "C:\ProgramData\Microsoft\VisualStudio\Packages\_Instances\*" | Select-Object -First 1
-    return Join-Path $instanceFolder.FullName "catalog.json"
+    $instanceFolder = "C:\ProgramData\Microsoft\VisualStudio\Packages\_Instances\" + (Get-VisualStudioProduct -ProductType "VisualStudio").InstanceId
+    return Join-Path $instanceFolder "catalog.json"
 }
 
-function Get-VisualStudioPath {
-    return (Get-VSSetupInstance | Select-VSSetupInstance -Product *).InstallationPath
-}
+function Get-VisualStudioProduct {
+    Param
+    (
+        [Parameter(Mandatory)]
+        [ValidateSet('VisualStudio','BuildTools')]
+        [String] $ProductType
+    )
 
-function Get-VisualStudioPackages {
-    return (Get-VSSetupInstance | Select-VSSetupInstance -Product *).Packages
+    if ($ProductType -eq "VisualStudio")
+    {
+        $VSSelectionType = "*Enterprise*"
+    }
+    elseif ($ProductType -eq "BuildTools")
+    {
+        $VSSelectionType = "*Build*"
+    }
+    return Get-VSSetupInstance | Select-VSSetupInstance -Product * | Where-Object -Property DisplayName -like $VSSelectionType
```

Comment:
Why do we need Select-VSSetupInstance -Product *?

Reply:
actually no reason, thank you, i'll remove it